### DISCUSSION TITLE
Allow to retrieve specific layer from source

### DISCUSSION THREAD

**Discussion by sharkAndshark** [maplibre/martin#1914] [Ideas]
Could client retrieve specific layer from source, like `http://0.0.0.0:3000/{source}/{layer}/{z}/{x}/{y}` ?  I find it's useful when I'm using PMTiles made by other team.
I think the API is perfect and intuitive by now, so actually I'm not sure this change is needed.

**Comment by nyurik** [comment]
This **could** be done, but I am still not certain why someone would need it?  Thinking it out here:
* It will only be relevant to tiles from MBTiles, PMTiles, and tiles from a PG function. Also, in the case of a PG function, this would result in a major inefficiency - PG may spent lots of resources generating all layers and later discarding all but one. Passing layer info to the backend for optimization could be done, but would require some generalized method i guess.
* this might not be very useful for layer merging - e.g. `/{source1},{source2}/{layer}/...` might not have much value -- perhaps `/{source}~{layer}/z/x/y` or `/{layer}@{source}/z/x/y` pattern instead? Source ID can only work with letters, numbers, `.` , `-`, and `_` -- so any other symbol might be ok.
* tilejson would need to be adjusted - this would function as a new synthetic "source", so should have a separate tilejson that also filters the tilejson's vector tile layer info.
